Test fails at linking stage

When following the instructions, the compilation fails:

```
$ mkdir build
$ cd build
$ cmake ..
-- The C compiler identification is GNU 8.2.1
-- The CXX compiler identification is GNU 8.2.1
-- Check for working C compiler: /usr/bin/cc
-- Check for working C compiler: /usr/bin/cc -- works
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Detecting C compile features
-- Detecting C compile features - done
-- Check for working CXX compiler: /usr/bin/c++
-- Check for working CXX compiler: /usr/bin/c++ -- works
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Configuring done
-- Generating done
-- Build files have been written to: /home/pk/Build/libecpint/build
$ make -j 4
[...]
[ 97%] Linking CXX static library libecpint.a
[ 97%] Built target ecpint
Scanning dependencies of target test1
Scanning dependencies of target test2
[ 98%] Building CXX object tests/test1/CMakeFiles/test1.dir/test1.cpp.o
[ 98%] Building CXX object tests/test2/CMakeFiles/test2.dir/test2.cpp.o
[100%] Linking CXX executable test2
[100%] Linking CXX executable test1
/usr/bin/ld: ../../src/libecpint.a(ecpint.cpp.o): in function `libecpint::ECPIntegral::type2(int, libecpint::ECP&, libecpint::GaussianShell&, libecpint::GaussianShell&, libecpint::ShellPairData&, libecpint::FiveIndex<double>&, libecpint::FiveIndex<double>&, libecpint::ThreeIndex<double>&)':
ecpint.cpp:(.text+0x1433): undefined reference to `libecpint::qgen::rolled_up(int, int, int, libecpint::ThreeIndex<double>&, libecpint::FiveIndex<double>&, libecpint::FiveIndex<double>&, libecpint::TwoIndex<double>&, libecpint::TwoIndex<double>&, libecpint::AngularIntegral&, libecpint::ThreeIndex<double>&)'
```
It looks as if the unrolled functions are not included in the library, for some reason. I wonder if I'm missing a config option.

Unfortunately, that doesn't seem to help. I get essentially the same error:
```
(p4dev36) pk@dream:~/Build/libecpint/build::make test1
[  3%] Built target generate
[ 98%] Built target ecpint
[100%] Linking CXX executable test1
../../src/libecpint.a(ecpint.cpp.o): In function `libecpint::ECPIntegral::type2(int, libecpint::ECP&, libecpint::GaussianShell&, libecpint::GaussianShell&, libecpint::ShellPairData&, libecpint::FiveIndex<double>&, libecpint::FiveIndex<double>&, libecpint::ThreeIndex<double>&)':
ecpint.cpp:(.text+0x1602): undefined reference to `libecpint::qgen::rolled_up(int, int, int, libecpint::ThreeIndex<double>&, libecpint::FiveIndex<double>&, libecpint::FiveIndex<double>&, libecpint::TwoIndex<double>&, libecpint::TwoIndex<double>&, libecpint::AngularIntegral&, libecpint::ThreeIndex<double>&)'
```
Despite the qgen object file containing the function:
```
(p4dev36) pk@dream:~/Build/libecpint/build::nm -C src/CMakeFiles/ecpint.dir/lib/qgen.cpp.o 
                 U _GLOBAL_OFFSET_TABLE_
0000000000000000 r .LC0
0000000000000000 r .LC1
0000000000000008 r .LC2
0000000000000000 T libecpint::rolled_up(int, int, int, libecpint::ThreeIndex<double>&, libecpint::FiveIndex<double>&, libecpint::FiveIndex<double>&, libecpint::TwoIndex<double>&, libecpint::TwoIndex<double>&, libecpint::AngularIntegral&, libecpint::ThreeIndex<double>&)
                 U libecpint::AngularIntegral::getIntegral(int, int, int, int, int, int, int) const
```
However, the library itself shows a lot of undefined symbols (these are just first three:
```
(p4dev36) pk@dream:~/Build/libecpint/build::nm -C src/libecpint.a | grep rolled | head -n 3
                 U libecpint::qgen::rolled_up(int, int, int, libecpint::ThreeIndex<double>&, libecpint::FiveIndex<double>&, libecpint::FiveIndex<double>&, libecpint::TwoIndex<double>&, libecpint::TwoIndex<double>&, libecpint::AngularIntegral&, libecpint::ThreeIndex<double>&)
0000000000000000 T libecpint::rolled_up(int, int, int, libecpint::ThreeIndex<double>&, libecpint::FiveIndex<double>&, libecpint::FiveIndex<double>&, libecpint::TwoIndex<double>&, libecpint::TwoIndex<double>&, libecpint::AngularIntegral&, libecpint::ThreeIndex<double>&)
                 U libecpint::qgen::rolled_up(int, int, int, libecpint::ThreeIndex<double>&, libecpint::FiveIndex<double>&, libecpint::FiveIndex<double>&, libecpint::TwoIndex<double>&, libecpint::TwoIndex<double>&, libecpint::AngularIntegral&, libecpint::ThreeIndex<double>&)
```
I'm not very good with cmake (as you can tell), but I'm guessing something is not linking into the lib properly. Also, the "T" symbol is missing a `::qgen::` namespace. Perhaps a namespace issue?

--- a/src/lib/qgen.cpp
+++ b/src/lib/qgen.cpp
@@ -3,6 +3,7 @@
 #include "mathutil.hpp"
 
 namespace libecpint {
+        namespace qgen {
 	
   void rolled_up(int lam, int LA, int LB, ThreeIndex<double> &radials, FiveIndex<double> &CA, FiveIndex<double> &CB, TwoIndex<double> &SA, TwoIndex<double> &SB, AngularIntegral &angint, ThreeIndex<double> &values)
 	{
@@ -69,4 +70,5 @@ namespace libecpint {
 		}
 	}
 	
+}
 }

